packages/core/src/utils/ConstantsUtil.ts

```diff
@@ -61,7 +61,101 @@ export const ConstantsUtil = {
     Base: 'base'
   },
 
-  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e'
+  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e',
+
+  SUGGESTED_TOKENS: [
+    'ETH',
+    'UNI',
+    '1INCH',
+    'AAVE',
+    'SOL',
+    'ADA',
+    'AVAX',
+    'DOT',
+    'LINK',
+    'NITRO',
+    'GAIA',
+    'MILK',
+    'TRX',
+    'NEAR',
+    'GNO',
+    'WBTC',
+    'DAI',
+    'WETH',
+    'USDC',
+    'USDT',
+    'ARB',
+    'BAL',
+    'BICO',
+    'CRV',
+    'ENS',
+    'MATIC',
+    'OP'
+  ],
+
+  POPULAR_TOKENS: [
+    'ETH',
```

Comment:
Some list of tokens to show for the demo purpose. Waiting Lit's virtualize SSR support for listing all possible tokens in the performant way.

- https://github.com/lit/lit/pull/4564

Reply:
Would this be something that we could store in api.web3modal maybe?

Reply:
Can the list be fetched api side?

Reply:
Here, I'd like to not filter the possible tokens, instead list all of them. But to do this performantly, we need to use `lit-virtualizer` which is a package to render lots of items without performance issues.

But we have some SSR issues on lit-virtualizer side which is resolved but not merge yet: https://github.com/lit/lit/pull/4564